Dockerfile

```diff
@@ -36,6 +36,13 @@ RUN curl -sSLo /tmp/asset-devworkspace-operator.zip https://api.github.com/repos
     curl -sSLo /tmp/asset-devworkspace-che-operator.zip https://api.github.com/repos/che-incubator/devworkspace-che-operator/zipball/${DEV_WORKSPACE_CHE_OPERATOR_VERSION} && \
     curl -sSLo /tmp/asset-restic.zip https://api.github.com/repos/restic/restic/zipball/${RESTIC_TAG}
 
+RUN mkdir -p $GOPATH/restic && cd $GOPATH/restic && \
+    unzip /tmp/asset-restic.zip -d /tmp && \
+    mv /tmp/restic-*/* $GOPATH/restic && \
+    go mod vendor && \
+    rm -f /tmp/asset-restic.zip && \
+    zip -r /tmp/asset-restic.zip .
```

Comment:
Should we use -r9 to get more compression here?

Reply:
actually... do we even need to re-create the zip inside the container? could also grab the contents via a bootstrap build.

Will need to replace this code https://github.com/redhat-developer/codeready-workspaces-images/blob/crw-2-rhel-8/codeready-workspaces-operator/get-sources-jenkins.sh#L59-L74 with something that builds the che-operator container including the vendor folder anyway, so perhaps creating the restic zip is unneeded at this point.